README.md

```diff
@@ -52,3 +52,33 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+### create a place to store data for todo app
+
+*create a trello account
+
+*create a trello board to store todo app item data
+
+*create a "to do" and "done" list.(you will have the option to create a "doing" list but this can be left blank)
+
+*go to trell.com/api-key do access api key and token
+
+*get the to do and done list id's and the board id. I did this with hoptscotch 
+
+*to get board ID REQUIRES API TOKEN AND KEY- https://api.trello.com/1/members/me/boards?key=***KEY***&token=***TOKEN***.......
+
+*to get the list id's, MUST ADD VAVIABLES KEY, TOKEN, CARDS  https://api.trello.com/1/boards/***BoardID***/lists
```

Comment:
Just a typo; worth scanning for if you get a chance to catch things that might make it harder for future users to read
```suggestion
*to get the list id's, MUST ADD VARIABLES KEY, TOKEN, CARDS  https://api.trello.com/1/boards/***BoardID***/lists
```